portals/new_admin/portals/client/src/components/App.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <h1>Hello World0!</h1>
+    );
+  }
+}
+
+
+export default App;
```

Comment:
Shall we add an EOF new line here as well?

Reply:
Currently, there is no such file.